app/controllers/file_metadata_controller.rb

```diff
@@ -0,0 +1,46 @@
+# frozen_string_literal: true
+class FileMetadataController < ApplicationController
+  delegate :query_service, to: :change_set_persister
+
+  def new
+    authorize! :update, file_set_change_set.resource
+    @change_set = ChangeSet.for(FileMetadata.new, change_set_param: change_set_param)
+    render "new"
```

Comment:
Does this have to be explicit because of the routing base? Normally it would be inferred I think.

Reply:
Thank you! I'd completely forgotten 😂